config.json

```diff
@@ -87,4 +87,14 @@
     "sheet-id": "971719928",
     "slack-channel": "C047C0CDS15",
     "message": "Morning :wave: This weeks point person for DPC release comms is: <<1>>. \nPlease refer to this <https://matillion.atlassian.net/wiki/spaces/SETL/pages/3724313001/DPC+Release+Comms|Confluence page> as a reminder of the process.\n\nNext week, the point person will be <<2>>.If there is any issue with this, please be sure to arrange for cover. The rota can be found <https://docs.google.com/spreadsheets/d/199rauTQCN7B2AdsRUrFMdYlh_sX8wgdh0uZxklYnntk/edit#gid=822553857|here>"
-}]
\ No newline at end of file
+},
+{
+    "name": "DPC AI Team",
+    "description": "Rota for the DPC AI Team",
+    "owner": "Rick Gale",
+    "days-active": "YYYYY",
+    "spreadsheet": "1T0cOPUfKsW6aK5Pswarku_bdYr6m73F0lRkVnO_cDJs",
```

Comment:
You've got a different value here than in the message:
* https://docs.google.com/spreadsheets/d/1kA8S5GBTHn6THwEv2mjx3If75DlYhHP4oONSJEHmpi4 (Message - "AI Team Rota")
* https://docs.google.com/spreadsheets/d/1T0cOPUfKsW6aK5Pswarku_bdYr6m73F0lRkVnO_cDJs (Configuration -"SaaS ETL Rota"
Identifies a potential improvement though to remove this redundancy and have it place this in itself :)

Reply:
Ah good spot! I think I've addressed this now.

Reply:
Is that the right one you've selected? I thought your sheet would have been https://docs.google.com/spreadsheets/d/1kA8S5GBTHn6THwEv2mjx3If75DlYhHP4oONSJEHmpi4/edit#gid=1162881823?